2019Labs/RHELSecurityLab/documentation/lab1_OpenSCAP.adoc

```diff
@@ -74,36 +77,52 @@ Notice that this command outputs the OpenSCAP version and versions of supported
+For Red Hat Enterprise Linux 8, OSPP (Protection Profile for General Purpose Operating Systems) and PCI-DSS (Payment Card Industry Data Security Standard) profiles are available.
+There are also various formats in which this is provided - human readable HTML guides, SCAP benchmarks, but also Ansible remediation playbooks.
 
 . Move to the `content` folder so we can avoid typing long paths:
 +
+----
  [root@openscap]# cd /usr/share/xml/scap/ssg/content
+----
 
 . First, let's check which compliance profiles are available for rhel8.
 +
 ----
  [root@openscap]# oscap info ssg-rhel8-ds.xml
 ----
-You will get print out of contents - aforementioned two compliance profiles. OSPP - Protection Profile for General Purpose Operating Systems, and PCI-DSS..
-
- . Next, let's perform our first baseline testing scan with the vanilla OSPP profile. Notice in our command below that we can skip the profile ID prefix to make the command simpler. The real ID is xccdf_org.ssgproject.content_profile_ospp.
++
+. You will get list of contents, aforementioned compliance profiles. OSPP - Protection Profile for General Purpose Operating Systems, and PCI-DSS.
 
+. Next, let's perform our first baseline testing scan with the vanilla OSPP profile.
+You can see in our command below that we can skip the profile ID prefix to make the command simpler.
+The real ID is xccdf_org.ssgproject.content_profile_ospp.
++
+----
   [root@openscap]# oscap xccdf eval --profile ospp ./ssg-rhel8-ds.xml
+----
 +
 Now, you will see the compliance scan results for every security control in the OSPP security baseline profile.
 
 . Now, let's store the results of the scan this time:
-* use --results-arf to get machine readable results archive
-* use --report to get human readable report (can also be generated from ARF after the scan as you see in the next optional step)
+* use `--results-arf` to get machine readable results archive
+* use `--report` to get human readable report (can also be generated from ARF after the scan as you see in the next optional step)
+* use `--oval-results` for extended details of failing rules
 
- [root@openscap]# oscap xccdf eval --profile ospp --results-arf /tmp/arf.xml --report /tmp/report.html ./ssg-rhel8-ds.xml
+----
+ [root@openscap]# oscap xccdf eval --oval-results --profile ospp --results-arf /tmp/arf.xml --report /tmp/report.html ./ssg-rhel8-ds.xml
+----
++
 
 . (Optional) You can also generate the HTML report separately:
 +
+----
  [root@openscap]# oscap xccdf generate report /tmp/arf.xml > /tmp/report.html
+----
 
-. Now, go back to your *Lab Information* webpage from the *Lab 0 setup steps* and click on the console button for your workstation bastion host. Login as *lab-user* with *r3dh4t1!* as the password.
+. Now, go to your *power controls and consoles* view from the *Lab Information* page (the one with GUID) and click on the console button for your workstation bastion host.
```

Comment:
probably a link would be better

Reply:
Yes. But link is dynamic, so I cannot really put it here.

Reply:
aha ok